tests/unit/test_views.py

```diff
@@ -0,0 +1,68 @@
+from unittest.mock import ANY
+from unittest.mock import Mock
+
+import pytest
+
+from footbot.main import app
+from footbot.main import optimise_team_route
+
+
+@pytest.fixture
+def flask_app():
+    yield app
+
+
+def test_optimise_team_route_public(flask_app):
```

Comment:
So here it looks like we're testing that the arguments are passed to the `optimise_entry` function correctly, which has been replaced by a mocked version. Is that right?

Reply:
yes- we're unit testing the handler / "view". since this (nicely-designed 👍) function only responsibility is to parse out parameters from the request and call the `optimise_entry` function with them,  we just give it some request and check that it gets the correct args